web/static/js/components/channels/ChannelStatus.jsx

```diff
@@ -14,7 +14,7 @@ class ChannelStatus extends Component {
     let info
     switch (statusInfo.status) {
       case 'down':
-        info = ` (${statusInfo.messages}) `
+        if (typeof statusInfo.messages !== 'undefined' && statusInfo.messages.trim() !== '') info = ` (${statusInfo.messages}) `
```

Comment:
Did you try `if (statusInfo.messages && statusInfo.messages.trim() !== '')` instead?

Reply:
Good catch, check now